Packs/PAN-OS/Playbooks/PAN-OS_-_Block_IP.yml

```diff
@@ -0,0 +1,517 @@
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    message:
+      to:
+      subject:
+      body:
+      methods: []
+      format: ""
+      bcc:
+      cc:
+      timings:
+        retriescount: 2
+        retriesinterval: 360
+        completeafterreplies: 1
+        completeafterv2: true
+        completeaftersla: false
+    form:
+      questions:
+      - id: "0"
+        label: ""
+        labelarg:
+          simple: Choose the action
+        required: true
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - simple: Do nothing
+        - simple: Add IP to static group
+        - simple: Tag the IP
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      - id: "1"
+        label: ""
+        labelarg:
+          simple: If you chose "Add to static group", please choose the relevant group
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: singleSelect
+        options: []
+        optionsarg:
+        - simple: N/A
+        - complex:
+            root: Panorama.AddressGroups
+            filters:
+            - - operator: isEqualString
+                left:
+                  value:
+                    simple: Panorama.AddressGroups.Type
+                  iscontext: true
+                right:
+                  value:
+                    simple: static
+                ignorecase: true
+            accessor: Name
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      - id: "2"
+        label: ""
+        labelarg:
+          simple: If you chose "Tag the IP", please choose the relevant tags
+        required: false
+        gridcolumns: []
+        defaultrows: []
+        type: multiSelect
+        options: []
+        optionsarg:
+        - simple: ${Panorama.Tag.name}
+        fieldassociated: ""
+        placeholder: ""
+        tooltip: ""
+        readonly: false
+      title: BlockIPAction
+      description: |-
+        This task will define the action which will be done on the enumerating IPs. Note that chosen actions will be automatically committed.
+        1. Do nothing - the playbook will continue without blocking the enumerating IP.
+        2. Add IP to static group - Choosing this also requires to choose the relevant address group in the 3rd question. The playbook will add the internal IP to the chosen address group. Please make sure to choose an address group that is configured in a relevant policy rule.
+        3. Tag the IP - Choosing this also requires to choose the relevant tag in the 3rd question. The playbook will create the IP address object (in case it doesn't exist) and will tag it with the chosen tag. Please make sure to choose a tag which is configured in a relevant dynamic address group or any other method which depends on the relevant tag.
```

Comment:
```suggestion
        3. Tag the IP - Choosing this option also requires you to choose the relevant tag in the 3rd question. The playbook will create the IP address object (in case it doesn't exist) and will tag it with the chosen tag. Please make sure to choose a tag which is configured in a relevant dynamic address group or any other method that depends on the relevant tag.
```